language-server/modules/langserver-commons/src/main/java/org/ballerinalang/langserver/commons/service/spi/ExtendedLanguageServerService.java

```diff
@@ -73,7 +73,9 @@ default void init(LanguageServer langServer,
      * Callback when client connected.
      *
      * @param client {@link ExtendedLanguageClient}
+     * @deprecated use {@link #init(LanguageServer, WorkspaceManager)}} instead.
```

Comment:
Shall we say `use LanguageServerContext` to obtain the `ExtendeLanguageClient` instead?

Reply:
+1